test/test_mps.py

```diff
@@ -9351,9 +9922,51 @@ def test_lstm_forward(self, device="mps", dtype=torch.float32):
                 self._lstm_helper(num_layers=num_layers, dtype=dtype, device=device, **test_options)
 
     def test_lstm_backward(self, device="mps", dtype=torch.float32):
-        for num_layers in [1] if product_version < 13.0 else [1, 2, 5]:
-            for test_options in self.LSTM_TEST_CASES:
-                self._lstm_helper(num_layers=num_layers, dtype=dtype, device=device, backward=True, **test_options)
+        for layers in [1] if product_version < 13.0 else [1, 2, 5]:
```

Comment:
Why are these one-off tests updated here? Any reason the LSTM OpInfo is not working for you?

Reply:
Good catch. This was a cherry-pick error from 2.0 release.